pkg/datapath/loader/config.go

```diff
@@ -12,6 +13,10 @@ import (
 func nodeConfig(lnc *datapath.LocalNodeConfiguration) config.Node {
 	node := *config.NewNode()
 
+	if option.Config.EnableIPv4 {
```

Comment:
I'd hoped the `CiliumInternalIPv6` would've inspired folks to take the same approach. :wink: All addresses can (should!) be turned into arrays now to avoid issues with endianness, unnecessary conversions and ambiguity.

Reply:
Changed in latest push.